substrate/frame/referenda/src/types.rs

```diff
@@ -139,7 +139,7 @@ pub struct TrackInfo<Balance, Moment> {
 /// Information on the voting tracks.
 pub trait TracksInfo<Balance, Moment> {
 	/// The identifier for a track.
-	type Id: Copy + Parameter + Ord + PartialOrd + Send + Sync + 'static + MaxEncodedLen;
+	type Id: Copy + Parameter + Ord + PartialOrd + Send + Sync + 'static + MaxEncodedLen + Default;
```

Comment:
`Id` -----> tracks, tracks -----> classes in `pallet_referenda`'s implementation of the `Polling` trait.

https://github.com/paritytech/polkadot-sdk/blob/c987da33935898cd5b2f8605d548bc48727c1815/substrate/frame/referenda/src/lib.rs#L750-L752

Reply:
This is a breaking change (major bump in the prdoc), since it can break existing code.  
If it is something that is only needed for benchmarking, then the benchmark can use a where clause to only work if the Id is Default, but not requiring it on the trait itself. (the trait does not care about benchmarks).

For example like here:  
https://github.com/paritytech/polkadot-sdk/blob/2ab3f03f0bb1773c3799997d541b60e6be5f67bd/bridges/snowbridge/pallets/outbound-queue/src/benchmarking.rs#L19